lib/sparkperf/testsuites.py

```diff
@@ -251,8 +251,8 @@ class MLlibTests(JVMPerfTestSuite, MLlibTestHelper):
     test_jar_path = "%s/mllib-tests/target/mllib-perf-tests-assembly.jar" % PROJ_DIR
 
     @classmethod
-    def build(cls):
-        run_cmd("cd %s/mllib-tests; %s clean assembly" % (PROJ_DIR, SBT_CMD))
+    def build(cls, spark_version):
+        run_cmd("cd %s/mllib-tests; %s -Dspark.version=%s.0 clean assembly" % (PROJ_DIR, SBT_CMD, spark_version))
```

Comment:
This means we'll always use the initial release, not follow-up patch versions.  I'd prefer to use patch versions if it's not too much trouble.